test/handshake.test.ts

```diff
@@ -1,42 +1,42 @@
-import { Server, Client, NodeClientOptions, ServerSocketStatus, ClientSocketStatus } from '../dist/index';
-import * as test from 'tape';
-import { create } from '../dist/lib/Util/Header';
+import { Server, Client, NodeClientOptions, ServerSocketStatus, ClientSocketStatus } from '../src/index';
+import { create } from '../src/lib/Util/Header';
 import { get, createServer } from 'http';
 import { serialize } from 'binarytf';
 import { URL } from 'url';
 import { readFileSync } from 'fs';
 
 let port = 8000;
 
-test('Basic Server', { timeout: 5000 }, async t => {
-	t.plan(5);
+beforeEach;
+test('Basic Server', async (done) => {
+	expect.assertions(5);
 
 	const nodeServer = new Server('Server');
 
 	try {
 		await nodeServer.listen(++port);
 	} catch (error) {
-		t.error(error, 'Server should not crash.');
+		expect(error).toBeFalsy();
 	}
 
 	// Connected
-	t.equal(nodeServer.sockets.size, 0, 'The server should not have a connected client.');
-	t.equal(nodeServer.name, 'Server', 'The server should be named after the node.');
+	expect(nodeServer.sockets.size).toBe(0);
+	expect(nodeServer.name).toBe('Server');
 
 	try {
 		await nodeServer.listen(++port);
-		t.fail('This call should definitely crash.');
+		done.fail('This call should definitely crash.');
```

Comment:
done.fail is not a function in Jest, that's an incorrect transformation. Remove it.